### PR TITLE
Project finish

### PR DESCRIPTION
**Pull request by davitomix**
- Changed background colors.
- Html validated.
- CSS validated.

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+linters:
+  csslint:
+    ignore: 'box-model,ids'
+files:
+  ignore: 'bower_components/*'
```
Comment (stratospherique) [inline .stickler.yml]: You are customizing the linter configuration. You should first follow the regular linters configuration. Check Micoroverse's [repo](https://github.com/microverseinc/linters-config/tree/master/css)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Align all elements with float OR flex AND your own Grid.
+Use article tag for all articles listed on the page.
+
+## Source
+https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown
+
+## Authors
+David and Gerald
+
+
```
Comment (stratospherique) [inline README.md]: - You can add links to your github repos
- A link to a live preview of the project

**Comment by stratospherique** [comment]
### That's it. When finished, kindly resubmit another code review request :pray:

**Comment by davitomix** [comment]
Ok thank you :)

**Comment by geraldgsh** [comment]
@stratospherique Thanks man! The stickler upped our CSS code formating!

**Pull request closed without merging**